tests/rptest/remote_scripts/stream_verifier_txn.py

```diff
@@ -0,0 +1,974 @@
+        start_webserver()
+    elif args.command in commands:
+        process_command(args.command, app_config, sys.stdout)
+    else:
+        data = {
+            'error':
+            f"topic swarm command "
+            f"'{args.command}' not yet implemented"
+        }
+        write_json(sys.stdout, data)
+        sys.exit(1)
+
+    return
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description=title)
+    parser.add_argument('-b',
+                        '--brokers',
+                        dest="brokers",
+                        default='localhost:9092',
+                        help="Bootstrap broker(s) (host[:port])")
+
+    parser.add_argument('-d',
+                        '--debug',
+                        dest="isdebug",
+                        action="store_true",
+                        default=False,
+                        help="Debug log level")
+
+    parser.add_argument('-w',
+                        '--workers',
+                        dest="worker_threads",
+                        default=4,
+                        help="Number of threads to process messages")
+
+    parser.add_argument(
+        "-f",
+        "--topic-prefix",
+        dest="topic_prefix",
+        default="topic-stream-verifier-tx",
+        help="Topic prefix to use when creating. "
+        "Formats: '<prefix>-p<partitions>-r<replicas>-<sequence_number>")
+
+    parser.add_argument("-g",
+                        "--group-id",
+                        dest="topic_group_id",
+                        default="group-stream-verifier-tx",
+                        help="Group it to use")
+
+    parser.add_argument('-c',
+                        '--topic-count',
+                        dest="topic_count",
+                        default=16,
+                        type=int,
+                        help="Number of topics to create")
+
+    subparsers = parser.add_subparsers(dest='command', required=True)
+    parser_produce = subparsers.add_parser(COMMAND_PRODUCE)
+
+    parser_produce.add_argument('--rps',
+                                dest='msg_rate',
+                                default=0,
+                                type=int,
+                                help="Producer's message rate per sec")
+    parser_produce.add_argument('-t',
+                                '--total',
+                                dest='msg_total',
+                                default=256,
+                                type=int,
+                                help="Producer's message rate per sec")
+
+    parser_consume = subparsers.add_parser(COMMAND_CONSUME)
+
+    parser_service = subparsers.add_parser(COMMAND_SERVICE)
+    parser_service.add_argument('--port',
+                                dest='web_port',
+                                default=8090,
+                                type=int,
+                                help="Webservice port to bind to")
+
+    main(parser.parse_args())
```

Comment:
nit: this is very unlikely to be used via CLI and has 150 LOC, wondering if we should get rid of this to avoid code bloat. just have a simple web server running and let the users interact via REST api?

Reply:
I have a feeling that this will be shared with clients and have such flexibility is a plus. Especially when writing scripts for some specific cases.